test/test-fork.c

```diff
@@ -636,6 +636,7 @@ static void assert_run_work(uv_loop_t* const loop) {
 }
 
 
+#ifndef __MVS__
 TEST_IMPL(fork_threadpool_queue_work_simple) {
```

Comment:
Shouldn't there be a similar `#ifndef __MVS__` in test/test-list.h?

Reply:
Fixed